services/worker-manager/schemas/v1/config-aws.yml

```diff
@@ -25,14 +25,27 @@ properties:
             ImageId:
               type: string
               description: The ID of the AMI to launch an instance from.
+            # These properties are not allowed, so we assert that
+            # they do not match the empty schema
+            TagSpecifications: {not: {}}
```

Comment:
why `TagSpecifications`?

Reply:
Looking at [this code](https://github.com/taskcluster/taskcluster/blob/8a4d49f3e8b134168781d84d4abf481774ffc6ae/services/worker-manager/src/providers/aws.js#L85) it appears that the `TagSpecification` is read from the config, not the `launchConfig`, and `TagSpecifications` is overridden in the `runInstances` call. So this will prevent it being specified and then ignored.

Reply:
That's a bug, Dustin and I are both aware of it and there's a fix coming

Reply:
Ok. That seemed like it might have been expected behavior. I'm happy to remove that, or it could be removed along with the fix to the bug?